spec/attime_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'attime'
+
+describe '#attime' do
```

Comment:
I think this should be:

```ruby
describe CheckGraphite do
  describe '.attime' do
```

or 

```ruby
describe 'CheckGraphite.attime' do
```

I.e. the description should include the module/class name, and in Ruby you refer to class/module methods as `.foo` and only instance methods as `#foo`.